notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb

```diff
@@ -653,7 +695,7 @@
       "source": [
         "### Get test item\n",
         "\n",
-        "You will use an arbitrary example out of the dataset as a test item. Don't be concerned that the example was likely used in training the model -- we just want to demonstrate how to make a prediction."
+        "You use an arbitrary example out of the dataset as a test item. Don't be concerned that the examples were likely used in training the model -- the purpose here's to demonstrate how to make a prediction."
```

Comment:
Do not contract "here is".
- change: "the purpose here's to ..."
- to: "the purpose here is to ..."